frontend/src/deviceDetailsPage/DeviceDetailsPage.jsx

```diff
@@ -41,7 +41,7 @@ export default function DeviceDetailsPage(props) {
             </Button>
           </Grid>
           <Grid item xs={2}>
-            <DeleteDeviceButton deleteId={device.serialNumber} />
+            <DeleteDeviceButton button deleteId={device.serialNumber} />
```

Comment:
I'm assuming that the button's boolean value is true by default?

Reply:
yep! and if i try to write `button={true}` the linter tells me to go to hell :)

Reply:
"yep, this do be a button!!!!"